homework/20/politicion-beans/main/java/com/example/LeftPolititions.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public class LeftPolititions extends Politition{

```

Comment:
It's spelled politician, I think :) 

Also, it'd be great if you have space before `{` so that it's consistent everywhere